src/Storefront/Framework/AffiliateTracking/AffiliateTrackingListener.php

```diff
@@ -13,6 +14,7 @@
 
 /**
  * @internal
+ * @deprecated tag:v6.8.0 - Will be removed in v6.8.0. Use cookies for affiliate and campaign tracking instead.
```

Comment:
```suggestion
 * @deprecated tag:v6.8.0 - Will be removed in v6.8.0. Use cookies for affiliate and campaign tracking instead. - reason:remove-subscriber
```

and remove the `Feature::triggerDeprecationOrThrow` calls again